base/codeintel-db/README.md

```diff
@@ -0,0 +1,8 @@
+## Important Notice
+
+Segment's self-hosted version of Sourcegraph uses an [External RDS Service](https://docs.sourcegraph.com/admin/external_services/postgres#kubernetes) for
+the `codeintel-db`.
+
+You can find the definition for that database in the [segmentio/terracode-tooling](https://github.com/segmentio/terracode-tooling) repository.
```

Comment:
In the two README links can you point to the specific file?

Reply:
Sure, I thought about that and opted to just leave it generic but since you called it out I can 👍

Reply:
Done in https://github.com/segmentio/deploy-sourcegraph/pull/14/commits/d90cd82cc0da165d014ff67a9e0165612ba0179d